src/us/kbase/auth2/providers/GoogleIdentityProviderFactory.java

```diff
@@ -53,29 +53,35 @@ public IdentityProvider configure(final IdentityProviderConfig cfg) {
 	 */
 	public static class GoogleIdentityProvider implements IdentityProvider {
 	
+		/* might want to take another crack at using the google client. Last time
```

Comment:
Hmmm...yeah, the Google example code for Java seems to all use the Google client.